core/iwasm/aot/aot_reloc.h

```diff
@@ -136,6 +136,8 @@ typedef struct {
     REG_SYM(truncf),                      \
     REG_SYM(rint),                        \
     REG_SYM(rintf),                       \
+    REG_SYM(sqrt),                        \
+    REG_SYM(sqrtf),                       \
```

Comment:
sqrt/sqrtf might be a macro but not function, adding them here may cause compile error, see Windows CI report:
https://github.com/bytecodealliance/wasm-micro-runtime/actions/runs/3406870901

Suggest to register them like aot_memset/memmove:
Declare and implement aot_sqrt and aot_sqrtf in aot_runtime.h and aot_runtime.c, just after the code of aot_memset and aot_memmove:
https://github.com/bytecodealliance/wasm-micro-runtime/blob/main/core/iwasm/aot/aot_runtime.h#L448-L452

And here:
```C
{ "sqrt", (void*)aot_sqrt },
{ "sqrtf", (void*)aot_sqrtf },
```

Reply:
Thanks. then how can we know which APIs may not function but macro? if in future we encounter similar problems again